### ISSUE TITLE
Run _kompile in parallel when `--with-llvm-library` is present

### ISSUE THREAD

**Issue by anvacaru** [runtimeverification/evm-semantics#1971]
Perhaps, it is worth using the `multiprocessing` lib on the two calls to `_kompile` [here].(https://github.com/runtimeverification/evm-semantics/blob/master/kevm-pyk/src/kevm_pyk/foundry.py#L398)

**Comment by ehildenb** [comment]
I think instead we should ask the K frontend team to just always compile the haskell backend with the booster and build the LLVM library for it as well. The booster backend should become the default.

**Comment by anvacaru** [comment]
closing this as duplicate of #1939.
